src/main/java/mate/academy/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
+        <File name = "LogToFile" fileName = "logs/app.log">
+            <PatternLayout>
+                <Pattern>%d %p %c:%L %m%n</Pattern>
+            </PatternLayout>
+        </File>
+    </Appenders>
+    <Loggers>
+        <Logger name = "mate.academy" level = "info" additivity = "false">
+            <AppenderRef ref = "LogToFile"/>
+            <AppenderRef ref = "Console"/>
+        </Logger>
+        <Root level="error">
+            <AppenderRef ref = "LogToFile"/>
+            <AppenderRef ref = "Console"/>
+        </Root>
+    </Loggers>
+</Configuration>
```

Comment:
have you tested your solution properly? Is logging correctly working? I have doubts because of the location of log4j2.xml file

Reply:
Is it should be in another package?